### ISSUE TITLE
ERR500 GET http://localhost:4000/api/v1/contacts/whatever

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#527]
It should not happens in a normal flow but there it is if someone tries to pass whatever instead of an id:

```
{
	"errors": [{
		"code": 500,
		"description": "The server has either erred or is incapable of performing the requested operation.",
		"component": "contact",
		"values": 500,
		"message": "None",
		"property": "HTTPInternalServerError",
		"type": "Internal Server Error"
	}]
}
```

**Comment by iamdey** [comment]
it's non-blocking

**Comment by sapiens-sapide** [comment]
Fix by commit #e821b35
